src/Components/animation/AnimationEditor.js

```diff
@@ -50,14 +48,43 @@ class AnimationEditor extends React.Component {
   };
 
   onLayout = e => {
+    var cHeight = e.nativeEvent.layout.height;
+    var cWidth = e.nativeEvent.layout.width;
+
     //TODO see why this is needed...
     this.setState({
-      width: e.nativeEvent.layout.width,
-      height: e.nativeEvent.layout.height,
-      dTop: 40,
+      width: cWidth,
+      height: cHeight,
+      dTop: e.nativeEvent.layout.y,
       dLeft: e.nativeEvent.layout.x,
     });
 
+    // Create the elements in the horizontal bar
+    this.initialElements = [];
+
+    var aWidth = cWidth * this.wRatio;
+    var aHeight = cHeight * this.hRatio;
+
+    debug('onLayout: aWidth/aHeight: ' + aWidth + '/' + aHeight);
+
+    var playerRadius = Math.min(aWidth, aHeight) / 12;
+    this.ddeTop = aHeight + 2.5 * playerRadius;
```

Comment:
dde only stands for https://fr.wikipedia.org/wiki/Direction_d%C3%A9partementale_de_l%27%C3%89quipement in my mind

Maybe use a more explicit variable name?

Reply:
DDE was for displayed draggable element ^^'. I replaced it.